hazelcast/src/main/java/com/hazelcast/cluster/Address.java

```diff
@@ -161,14 +165,12 @@ public boolean equals(Object o) {
             return false;
         }
         final Address address = (Address) o;
-        return port == address.port && this.type == address.type && this.host.equals(address.host);
+        return this.type == address.type && this.socketAddr.equals(address.socketAddr);
```

Comment:
`this.type == address.type`, do we need it?

Reply:
:+1: We don't need it. Nice spot.